### PR TITLE
CLN: replace %s syntax /w .format in plotting, stats, tseries

### PR DESCRIPTION
**Pull request by corazzon**
Progress towards #16130

- [x] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`

### CODE REVIEW

File: pandas/_version.py [hunk 1]
```diff
@@ -75,19 +75,20 @@ def run_command(commands, args, cwd=None, verbose=False, hide_stderr=False):
...
         if verbose:
-            print("unable to find command, tried %s" % (commands,))
+            print("unable to find command, tried {commands}".format(
+                commands=commands,))
         return None
     stdout = p.communicate()[0].strip()
     if sys.version_info[0] >= 3:
         stdout = stdout.decode()
     if p.returncode != 0:
         if verbose:
```
Comment (jreback) [inline pandas/_version.py]: revert this file it is auto generated

Comment (jreback) [reply pandas/_version.py]: rather versioneer creates it; would rather use a newer versioneer

**Comment by jreback** [comment]
can u rebase

File: pandas/tseries/offsets.py [hunk 2]
```diff
@@ -265,6 +265,22 @@ def _params(self):
         params = tuple([str(self.__class__)] + attrs)
         return params
 
+    def __repr__(self):
```
Comment (jreback) [inline pandas/tseries/offsets.py]: remove this; its a rebase artifact

**Review by jreback — CHANGES_REQUESTED** [review]

**Review by jreback — APPROVED** [review]

**Comment by jreback** [comment]
thanks!

**Comment by corazzon** [comment]
thanks!

**Pull request merged**